tools/build_and_test_onnxrt.sh

```diff
@@ -31,7 +31,7 @@ pip3 install -r requirements-dev.txt
 # Add newer cmake to the path
 export PATH="/opt/cmake/bin:$PATH"
 export CXXFLAGS="-D__HIP_PLATFORM_AMD__=1 -w"
-./build.sh --config Release  --cmake_extra_defines CMAKE_HIP_COMPILER=/opt/rocm/llvm/bin/clang++ --update --build --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --skip_tests --rocm_home /opt/rocm --use_migraphx --migraphx_home /opt/rocm --rocm_version=`cat /opt/rocm/.info/version-dev` --allow_running_as_root
+./build.sh --config Release  --cmake_extra_defines CMAKE_HIP_COMPILER=/opt/rocm/llvm/bin/clang++ --update --build --build_wheel --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --cmake_extra_defines FETCHCONTENT_TRY_FIND_PACKAGE_MODE=NEVER --skip_tests --rocm_home /opt/rocm --use_migraphx --rocm_version=`cat /opt/rocm/.info/version-dev` --allow_running_as_root
```

Comment:
We shouldn't be pulling down dependencies when we are building. Cant we just install the gmock package in the dockerfile?

Reply:
This was related to when using our internal branches which was missing a set of changes. I'll need to reconfirm with the ORT ROCm EP guys to see if this is fixed.